test/e2e/proza-message.test.js

```diff
@@ -8,76 +7,82 @@ describe('vl-proza-message', async () => {
     before(() => {
         return vlProzaMessagePage.load();
     });
+    
+    it('als gebruiker kan ik een tekst wijzigen door op het potlood te klikken', async () => {
```

Comment:
Best de tekst ook eens wijzigen en voor en na wijziging assert doen?

Reply:
Word volgens mij al in een andere test afgetoetst

Reply:
Ik pas de beschrijving aan zodat het duidelijker is dat er enkel het editeerbaar maken wordt getest